iceoryx_posh/source/popo/wait_set.cpp

```diff
@@ -31,58 +33,52 @@ WaitSet::WaitSet(cxx::not_null<ConditionVariableData* const> condVarDataPtr) noe
 
 WaitSet::~WaitSet() noexcept
 {
-    // Notify all conditions in the vector that the condition variable data will be destroyed
     detachAllConditions();
     /// @todo Notify RouDi that the condition variable data shall be destroyed
```

Comment:
we should take care of this TODO in #341

Reply:
See issue: https://github.com/eclipse/iceoryx/issues/369